core/node/consensus/src/registry/abi.rs

```diff
@@ -0,0 +1,225 @@
+//! Strongly-typed API for ConsensusRegistry contract.
+#![allow(dead_code)]
```

Comment:
Is it still needed?

Reply:
yes, most of the methods are used in tests only (deployment and writing calls)